### PR TITLE
Successful mnemonic verification

### LINKED ISSUE #105: Nothing happens after successful signing key mnemonic verification
Should happen:
   - save private key in redux state
   - navigate to withrawal key flow
   - remove verification route from history (back button should lead to generate mnemonic)

### PR DESCRIPTION
**Pull request by morrigan**
Resolves #105 
with created Withdrawal key initial step.

### CODE REVIEW

**Comment by morrigan** [comment]
@mpetrunic I've realized that I'm missing paragraph that says `If you skip, we won’t be able to generate a deposit transaction.` meaning that the skip button should go to create password screen?

**Comment by mpetrunic** [comment]
@morrigan yes

File: src/renderer/components/Button/ButtonStandard.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ export interface IButtonProps {
     onClick?: () => void;
     large?: boolean;
     buttonId?: string;
+    dataField?: any;
```
Comment (mpetrunic) [inline src/renderer/components/Button/ButtonStandard.tsx]: don't use any type. You could do something like `string|number`

Comment (mpetrunic) [inline src/renderer/components/Button/ButtonStandard.tsx]:
```suggestion
    dataField?: string | number;
```

**Comment by morrigan** [comment]
@mpetrunic thanks for the fix! :dancer:

**Pull request merged**